client/styles/vendors/_codemirror.scss

```diff
@@ -133,8 +133,6 @@
 
 /* Default styles for common addons */
 
-div.CodeMirror span.CodeMirror-matchingbracket {color: #0f0;}
```

Comment:
I think there's no need to modify this file, right? This is the default style file for CodeMirror, it will be overwritten by the custom style.